lib/cartopy/crs.py

```diff
@@ -1310,12 +1310,27 @@ def y_limits(self):
 class Stereographic(Projection):
     def __init__(self, central_latitude=0.0, central_longitude=0.0,
                  false_easting=0.0, false_northing=0.0,
-                 true_scale_latitude=None, globe=None):
+                 true_scale_latitude=None,
+                 scale_factor=None, globe=None):
         proj4_params = [('proj', 'stere'), ('lat_0', central_latitude),
                         ('lon_0', central_longitude),
                         ('x_0', false_easting), ('y_0', false_northing)]
+
         if true_scale_latitude:
```

Comment:
It's old, but should this be `is not None`? Since the `*PolarStereo` comments say defaults are not 0, it seems like you might want to pass a true scale latitude of 0, and that would be ignored with the code like this. Or is that basically useless since 0 is at the border?

Reply:
Of course you find that after I think I'm done. Checking for `None` makes sense to me since the latitude is float anyway--checking just for 0 is worthless.